DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -581,6 +588,117 @@ public Task<IReadOnlyList<DiscordInvite>> GetInvitesAsync()
+        /// Modifies a stage topic
+        /// </summary>
+        /// <param name="topic">New topic of stage.</param>
+        /// <param name="privacy_level">New privacy level of stage.</param>
+        /// <param name="reason">Audit log reason</param>
+        /// <exception cref="Exceptions.UnauthorizedException">Thrown when the client does not have the <see cref="Permissions.ManageChannels"/> permission.</exception>
+        /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
+        /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
+        /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        public async Task ModifyStageAsync(Optional<string> topic, Optional<StagePrivacyLevel> privacy_level, string reason = null)
+            => await this.Discord.ApiClient.ModifyStageInstanceAsync(this.Id, topic, privacy_level, reason);
+
+        /// <summary>
+        /// Closes a stage
+        /// </summary>
+        /// <param name="reason">Audit log reason</param>
+        /// <exception cref="Exceptions.UnauthorizedException">Thrown when the client does not have the <see cref="Permissions.ManageChannels"/> permission.</exception>
+        /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
+        /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
+        /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        public async Task CloseStageAsync(string reason = null)
+            => await this.Discord.ApiClient.DeleteStageInstanceAsync(this.Id, reason);
+
+        /// <summary>
+        /// Gets a stage
+        /// </summary>
+        /// <returns></returns>
+        /// <exception cref="Exceptions.UnauthorizedException">Thrown when the client does not have the <see cref="Permissions.AccessChannels"/> or <see cref="Permissions.UseVoice"/> permission.</exception>
+        /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
+        /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
+        /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        public async Task<DiscordStageInstance> GetStageAsync()
+            => await this.Discord.ApiClient.GetStageInstanceAsync(this.Id);
+
+        #endregion
+
+        #region Threads
+
+        /// <summary>
+        /// Creates a thread. Depending on whether it is created inside an <see cref="ChannelType.News"/> or an <see cref="ChannelType.Text"/> it is either an <see cref="ChannelType.NewsThread"/> or an <see cref="ChannelType.PublicThread"/>
```

Comment:
A thread within a thread?... That doesn't seem right.

Reply:
no this is discordchannel to create a public thread